stmhal/system_stm32.c

```diff
@@ -328,20 +345,23 @@ void SystemClock_Config(void)
      clocked below the maximum system frequency, to update the voltage scaling value
      regarding system frequency refer to product datasheet.  */
   __HAL_PWR_VOLTAGESCALING_CONFIG(PWR_REGULATOR_VOLTAGE_SCALE1);
-    #elif defined(MCU_SERIES_L4)
-    /* Enable the LSE Oscillator */
-    RCC_OscInitStruct.OscillatorType = RCC_OSCILLATORTYPE_LSE;
-    RCC_OscInitStruct.LSEState = RCC_LSE_ON;
-    if (HAL_RCC_OscConfig(&RCC_OscInitStruct) != HAL_OK) {
-        __fatal_error("HAL_RCC_OscConfig");
-    }
-    #endif
```

Comment:
It looks like the changes here change the behaviour of the L4 code.  I see that HAL_RCC_OscConfig is called twice for L4 (once here and once below) and I'm not sure if that was intended or not but the code is known to work and so it shouldn't be changed.  So please leave this part as it was.  (Unless you have an L4 board and have tested this change, but even then it would be better to put such a change in a separate PR describing it.)